cpp/src/barretenberg/ecc/curves/bn254/scalar_multiplication/scalar_multiplication.test.cpp

```diff
@@ -482,7 +482,7 @@ TEST(scalar_multiplication, construct_addition_chains)
                                                                        bucket_counts,
                                                                        &bit_offsets[0],
                                                                        state.point_schedule,
-                                                                       num_points,
+                                                                       static_cast<uint32_t>(state.round_counts[0]),
```

Comment:
Right, so `num_points` is really the number of points in a round, and this is determined where `round_counts` is set in the call to `compute_wnaf_states`. This test is not a real test because there are no assertions or expectations expressed, and probably a proper test would have made this error clear long ago. Is that right?

Reply:
Moreover, the commented-out line [here](https://github.com/AztecProtocol/barretenberg/blob/beb8e470fc8b6cfa846bcc7f15ce2e79fb05e184/cpp/src/barretenberg/ecc/curves/bn254/scalar_multiplication/scalar_multiplication.test.cpp#L491) should not be there, and the timing output is misleading: the line
`std::cout << "scalar mul: " << diff.count() << "ms" << std::endl;`
should be
`info("construct addition chains: ", diff.count(), "ms");`
Would you please delete the commented out line and change the printing?

Reply:
Yes, one sec